### DISCUSSION TITLE
Found alsa -> Could NOT find ALSA (SteamOS)

### DISCUSSION THREAD

**Discussion by MvEerd** [polybar/polybar#2908] [Build Support] [answered]
When building polybar-git from AUR using `makepkg -si` I get the following output;
```
-- Checking for module 'alsa'
--   Found alsa, version 1.2.7.2
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find ALSA (missing: ALSA_INCLUDE_DIRS) (found version "1.2.7.2")
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/common/utils.cmake:61 (find_package_handle_standard_args)
  cmake/modules/FindALSA.cmake:10 (find_package_impl)
  cmake/libpoly.cmake:54 (find_package)
  CMakeLists.txt:49 (include)


-- Configuring incomplete, errors occurred!
See also "/home/deck/polybar-git/src/polybar/build/CMakeFiles/CMakeOutput.log".
See also "/home/deck/polybar-git/src/polybar/build/CMakeFiles/CMakeError.log".
==> ERROR: A failure occurred in build().
    Aborting...
```
    
    
Any ideas what I am missing? The error is confusing as it first says it found alsa, then immediately says it could not find ALSA

**Comment by patrick96** [comment, marked as answer]
So, it seems it can find the alsa library but then can't find some of the files required from the library. Does the file `/usr/include/alsa/asoundlib.h` exist on your system?

**Reply by MvEerd** [reply]
`/usr/include/alsa/asoundlib.h` does not exist on my system, nor does the `/usr/include/alsa/` folder
`sudo find -name asoundlib.h` returns no results either

**Reply by patrick96** [reply]
Weird...

Does the file `/usr/lib/pkgconfig/alsa.pc` exist on your system?

**Reply by MvEerd** [reply]
That does exist
```sh
❯ stat /usr/lib/pkgconfig/alsa.pc
  File: /usr/lib/pkgconfig/alsa.pc
  Size: 277       	Blocks: 8          IO Block: 4096   regular file
Device: 0,32	Inode: 68548       Links: 1
Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2023-02-05 16:03:25.499040396 +0100
Modify: 2022-07-09 12:01:38.000000000 +0200
Change: 2022-12-29 00:53:38.303333309 +0100
 Birth: 2022-12-29 00:53:38.303333309 +0100
```

**Reply by patrick96** [reply]
Then the installation of your `alsa-lib` package is broken because it should provide both `/usr/include/alsa/asoundlib.h` and `/usr/lib/pkgconfig/alsa.pc`

**Reply by MvEerd** [reply]
`pacman -S alsa-lib` made this error go away 👍🏻  
Thanks for taking the time to help debug this, much appreciated.

I'll try to resolve the further errors that I am getting after resolving this alsa error
